CMakeLists.txt

```diff
@@ -95,7 +95,7 @@ if(WITH_MOAB)
     include_directories(${MOAB_INCLUDE_DIRS})
     get_filename_component(MOAB_LIBRARY_DIRS ${MOAB_LIBRARY} PATH)
     if(APPLE)
-      link_directories(${MOAB_LIBRARY_DIRS}/lib)
+      link_directories(${MOAB_LIBRARY_DIRS})
```

Comment:
If this is true, then do we need the conditional?  What changed?

Reply:
I am not sure, I just know that on my `Mac` I need this.
it is hard to be sure it won't brake other `Mac` installation :(